norbit-webapp/components/dashboard/Dashboard.tsx

```diff
@@ -51,6 +52,7 @@ const Dashboard: React.FC = () => {
         {sensorConfig.temperature && <TemperatureComponent amplifyInstance={mockAmplify ? null : Amplify} />}
         {sensorConfig.light && <LightIntensityComponent amplifyInstance={mockAmplify ? null : Amplify} />}
         {sensorConfig.sound && <SoundLevelComponent amplifyInstance={mockAmplify ? null : Amplify} />}
+        <Map amplifyInstance={mockAmplify ? null : Amplify} />
```

Comment:
SensorConfig needs to be extend with `location`, and this line needs to be
```suggestion
        {sensorConfig.location && <Map amplifyInstance={mockAmplify ? null : Amplify} />}
```